.github/workflows/build-test.yml

```diff
@@ -95,6 +99,12 @@ jobs:
           (cd fmc && cargo make build)
           sccache --show-stats
 
+      # Clippy needs to build crates as part of the check, so do it after the
+      # build.
+      - name: Clippy lint check
+        run: |
+          cargo clippy -- -D warnings
```

Comment:
Change to `cargo clippy --all-targets -- -D warnings` to pick up tests

Reply:
Done